web/src/beta/features/Editor/tabs/map/BottomPanel/index.tsx

```diff
@@ -0,0 +1,53 @@
+import SidePanelCommon from "@reearth/beta/features/Editor/SidePanel";
+import {
+  LayerStyleAddProps,
+  LayerStyleNameUpdateProps,
+} from "@reearth/beta/features/Editor/useLayerStyles";
+import { LayerStyle } from "@reearth/services/api/layerStyleApi/utils";
+import { useT } from "@reearth/services/i18n";
+
+import LayerStyles from "./layerStyle";
+
+type Props = {
+  layerStyles: LayerStyle[];
+  selectedLayerStyleId?: string;
+  onLayerStyleAdd: (inp: LayerStyleAddProps) => void;
+  onLayerStyleDelete: (id: string) => void;
+  onLayerStyleNameUpdate: (inp: LayerStyleNameUpdateProps) => void;
+  onLayerStyleSelect: (id: string) => void;
+};
+
+const MapSidePanel: React.FC<Props> = ({
+  layerStyles,
+  selectedLayerStyleId,
+  onLayerStyleAdd,
+  onLayerStyleDelete,
+  onLayerStyleNameUpdate,
+  onLayerStyleSelect,
+}) => {
+  const t = useT();
+
+  return (
+    <SidePanelCommon
+      location="bottom"
+      contents={[
+        {
+          id: "layerStyle",
+          title: t("LayerStyle"),
```

Comment:
You have LayerStyle in a few places. There should be a space between them `Layer Style`